### DISCUSSION TITLE
drf-excel needs contributors (writing tests)

### DISCUSSION THREAD

**Discussion by tim-schilling** [django-commons/membership#50] [Announcements]
The [drf-excel](https://github.com/wharton/drf-excel) project built by Wharton is [looking to move to Django Commons](https://github.com/wharton/drf-excel/issues/86). First, though it needs some tests. If you're looking to get started contributing to open-source and becoming a maintainer, this is the perfect task for you. This code is used in production in many areas and is known to work well.

We could use your help!

**Comment by browniebroke** [comment]
Started to look at this here https://github.com/wharton/drf-excel/pull/87

**Reply by tim-schilling** [reply]
Thanks for getting that started!

**Comment by tim-schilling** [comment]
I'm closing this discussion as there are tests written. Thank you @browniebroke for your help!
